swan-lake/learn-the-platform/native-support/build-a-native-executable.md

```diff
@@ -349,3 +350,8 @@ After the environment is set up, follow the steps below to build the native exec
    $ curl http://localhost:8080/greeting
    Hello, Docker!
    ```
+   
+## Known issues
```

Comment:
```suggestion
## Known issues
```
Should we add known issues on the Learn page? @shafreenAnfar

Reply:
Yeah, we will be fixing these for the official release.